src/libraries/Microsoft.NETCore.Platforms/src/RID.cs

```diff
@@ -1,37 +1,44 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
+using System.Diagnostics;
 using System.Text;
 
 namespace Microsoft.NETCore.Platforms.BuildTasks
 {
-    internal class RID
+    public class RID
```

Comment:
I guess the access modifier changed because of tests depending on the RID class now?

Reply:
Yeah, I didn't bother setting up internals visible to since this is an msbuild task and the only consumer of it is our build and tests.